packages/cxl-ui/src/components/cxl-base-card.js

```diff
@@ -56,7 +56,7 @@ export class CXLBaseCardElement extends LitElement {
 
   _renderHeaderName() {
     return html`
-      <div class="name">
+      <div class="name" title="${this.name}">
```

Comment:
Small tidbit: attribute binding doesn't need encapsulating quotes.

```suggestion
      <div class="name" title=${this.name}>
```